### DISCUSSION TITLE
[A-L] (20 Settembre 2024) Esercizio 1

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#417] [Esami A-L]
Per ogni n naturale consideriamo l'insieme delle sequenze binarie di lunghezza n.

Rispondere alle seguenti domande esprimendo la risposta come funzione di n.

1. Quante sequenze binarie di lunghezza $\Large n \geq 4$ iniziano con 01 e finiscono con 11? [1 punto]

2. Quante sequenze binarie di lunghezza $\Large n \geq 2$ non iniziano e non finiscono con 1? [2 punti]

3. Per $\Large n=2k$ quante sequenze binarie di lunghezza n contengono lo stesso numero di 0 e 1? [2 punti]

**Comment by BenjiKunji** [comment]
1) Iniziano e finiscono con **01** e **11**. Le cifre in mezzo sono **n-4 cifre**. Per ogni cifra abbiamo 2 possibilità, 0 o 1.  Questo ci porta ad avere **2<sup>n-4</sup>** possibili stringhe.
2) Iniziano e finiscono con 0. Le cifre in mezzo sono **n-2 cifre**. Stesso ragionamento. Le possibili stringhe sono **2<sup>n-2</sup>**.
3) Per n = 2k, e vogliamo **k** zeri e **k** uni. Se trovo i modi di disporre esattamente k zeri in stringhe lunghe 2k cifre, le rimanenti cifre saranno necessariamente 1, e per fare ciò usiamo i coefficienti binomiali. La risposta è **$\binom{2k}{k}$**
